chpl/chpl-service/src/main/java/gov/healthit/chpl/scheduler/job/extra/JobResponseTriggerListener.java

```diff
@@ -0,0 +1,80 @@
+package gov.healthit.chpl.scheduler.job.extra;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.apache.logging.log4j.Logger;
+import org.quartz.JobExecutionContext;
+import org.quartz.Trigger;
+import org.quartz.Trigger.CompletedExecutionInstruction;
+import org.quartz.TriggerListener;
+import org.springframework.core.env.Environment;
+
+public class JobResponseTriggerListener implements TriggerListener {
+    private static final String TRIGGER_LISTENER_NAME = "UpdateSingleListingTriggerListener";
+    private Logger logger;
```

Comment:
Where does this log to? Should it be loading one of the ones configured in log4j? Also private/static/final?

Reply:
The logger gets passed in since it can used by any job that might use require a simple response.
```
public JobResponseTriggerListener(List<JobResponseTriggerWrapper> triggerWrappers, final 
            String email, final Environment env, final Logger logger) {
    this.triggerWrappers = triggerWrappers;
    this.email = email;
    this.env = env;
    this.logger = logger;
}

```